src/services/dashboards/all-dashboards/modules/AllDashboardsSelectFilter.vue

```diff
@@ -68,6 +72,7 @@ export default defineComponent({
             ]),
             viewersStatus: computed(() => store.state.dashboard.viewers),
             scopeStatus: computed(() => store.state.dashboard.scope),
+            pagePermission: store.getters['user/pagePermissionMap'],
```

Comment:
question: Does it work as reactive ? 😲
I thought the code below works only.
If it works, it would be far better than using like below 🤓

```typescript
            pagePermission: computed(() => store.getters['user/pagePermissionMap'].value),
```